### PR TITLE
Made sure the .config.bundle is initialized correctly and thus a corr…

### PR DESCRIPTION
**Pull request by iuliana**
…ect URL is generated when a catalog item  is added from the composer

### CODE REVIEW

File: ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js [hunk 1]
```diff
@@ -101,9 +101,10 @@ export function saveToCatalogModalDirective($rootScope, $uibModal, $injector, $f
                     $scope.config.version = entity.version || metadata.get('version');
                 }
                 if (!$scope.config.bundle) {
-                    if ($scope.config.symbolicName) {
-                        $scope.config.bundle = $scope.config.symbolicName;
+                    if (!$scope.config.symbolicName) {
+                        $scope.config.symbolicName =  $scope.config.name;
```
Comment (algairim) [inline ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js:105]: There is a lot of logic tied up to `symbolicName` and to its absence. It effects the behaviour of c`omposer-blueprint-name-validator` directive a well, the absence of any tests makes it harder to see. I would not initialise `symbolicName` here but assign the `conig.name` to `config.bundle` directly, if `symbolicName` is not initialised.
`symbolicName` is initialized in couple of places already.

Comment (iuliana) [reply ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js:105]: `symbolicName` is initialized at line 98 already. - only it isn't, that is why the link is broken.

File: ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js [hunk 2]
```diff
@@ -142,6 +142,8 @@ export function CatalogItemModalController($scope, $filter, blueprintService, pa
         saving: false,
         force: false,
     };
+    $scope.catalogBundleId ="";
+    $scope.catalogBundleBase ="";
```
Comment (ahgittin) [inline ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js:146]: `""` will generate incorrect URL so not sure it's worth initializing with invalid value, but it should get populated quickly so NBD

**Comment by ahgittin** [comment]
LGTM - merging

**Pull request closed without merging**